docs/_static/quadratic_bowl.py

```diff
@@ -14,19 +14,19 @@ def quadratic_bowl(x, y):
 z_values = quadratic_bowl(x_values, y_values)
 fig = plt.figure()
 ax = fig.add_subplot(111, projection="3d")
-ax.plot_surface(x_values, y_values, z_values, cmap="coolwarm")
+ax.plot_surface(x_values, y_values, z_values, cmap="coolwarm")  # pyright: ignore
 plt.xticks([-10, -5, 0, 5, 10])
 plt.yticks([-10, -5, 0, 5, 10])
-ax.set_zticks([0, 50, 100, 150, 200])
+ax.set_zticks([0, 50, 100, 150, 200])  # pyright: ignore
 ax.set_xticklabels([])
 ax.set_yticklabels([])
-ax.set_zticklabels([])
+ax.set_zticklabels([])  # pyright: ignore
 ax.set_xlabel("x", labelpad=-28)
 ax.set_ylabel("y", labelpad=-28)
-ax.set_zlabel("f(x, y)\n\n\n\n\n\n\n\n", labelpad=-30)
-ax.zaxis.set_rotate_label(False)
-ax.xaxis.set_pane_color((1.0, 1.0, 1.0, 1.0))
-ax.yaxis.set_pane_color((1.0, 1.0, 1.0, 1.0))
-ax.zaxis.set_pane_color((1.0, 1.0, 1.0, 1.0))
+ax.set_zlabel("f(x, y)\n\n\n\n\n\n\n\n", labelpad=-30)  # pyright: ignore
+ax.zaxis.set_rotate_label(False)  # pyright: ignore
+ax.xaxis.set_pane_color((1.0, 1.0, 1.0, 1.0))  # pyright: ignore
+ax.yaxis.set_pane_color((1.0, 1.0, 1.0, 1.0))  # pyright: ignore
+ax.zaxis.set_pane_color((1.0, 1.0, 1.0, 1.0))  # pyright: ignore
```

Comment:
Nit: Can these be `pyright: ignore[somethingSpecific]`?

(I didn't used to do this, but I think it's a best practice.)